app/helpers/products_helper.rb

```diff
@@ -47,13 +47,15 @@ def set_sorting(test, test_status)
 
     case test_status
     when 'passing'
-      return 5
+      return 6
```

Comment:
Instead of returning `5`, return something like `TABLE_FIRST`, which is set to 0. I guess it makes a little more sense when I know that it's a sort order thing. Maybe just a comment to that effect would be sufficient.